src/gmt_support.c

```diff
@@ -16975,13 +16975,46 @@ bool gmtlib_script_is_classic (struct GMT_CTRL *GMT, FILE *fp) {
 	return (!modern);
 }
 
+int gmt_token_check (struct GMT_CTRL *GMT, FILE *fp, char *prefix, unsigned int mode) {
```

Comment:
Some potential issues:
1. Tried with anim01, but it reports some errors like
```
Main script uses MOVIE_FRAME} but without the required leading $
```
2. Comment lines or inline comments should be skipped
3. csh doesn't support `$()` syntax

Reply:
Addressed in the update